.github/workflows/scripts/setup_bundler

```diff
@@ -81,8 +81,7 @@ function install_ruby_version_specific_gems {
     gem install --default digest:3.1.0
     gem install --default digest:3.2.0
     gem install --default erb:4.0.4
-    gem install --default erb:5.0.1
-    gem install --default logger:1.7.0
```

Comment:
Instead of totally removing it, we should put this behind an if statement that checks the ruby version first, otherwise we'll see the issue we saw before with "i already activated a different version" the first time we run a test suite